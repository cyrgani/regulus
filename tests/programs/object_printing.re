type(
Foo,
a, bar, c
)

# TODO: unstable output, see `impl Display for Atom`
#print(Foo(4, "d", list(1, 4, Foo(0, null, true))))
