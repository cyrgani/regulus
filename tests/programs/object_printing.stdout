{a: 4, bar: d, c: [1, 4, {a: 0, bar: null, c: true}]}
